src/highlighters.cc

```diff
@@ -998,6 +998,89 @@ struct TabulationHighlighter : Highlighter
     }
 };
 
+struct ShowMatchingHighlighter : Highlighter
+{
+    ShowMatchingHighlighter(int offset)
+      : Highlighter{HighlightPass::Move}, m_offset{std::move(offset)}
```

Comment:
std::move on an int is unecessary and looks a bit weird. Also I think the offset should not be an int but a CharCount: we want to look at the previous codepoint.